.circleci/config.yml

```diff
@@ -111,6 +111,21 @@ jobs:
             pip install --user --progress-bar off jinja2 pyyaml
             python .circleci/regenerate.py
             git diff --exit-code || (echo ".circleci/config.yml not in sync with config.yml.in! Run .circleci/regenerate.py to update config"; exit 1)
+  
```

Comment:
@jdsgomes Did you directly edit `config.yml`? This file is generated with `.circleci/regenerate.py` file to reduce repetition. The change needs to go into `config.yml.in`.